db/stats.py

```diff
@@ -16,9 +16,9 @@
 STATS_CACHE_TIMEOUT = 60 * 10  # 10 minutes
 LAST_MBIDS_CACHE_TIMEOUT = 60  # 1 minute (this query is cheap)
 
-STATS_MEMCACHE_KEY = "recent-stats"
-STATS_MEMCACHE_LAST_UPDATE_KEY = "recent-stats-last-updated"
-STATS_MEMCACHE_NAMESPACE = "statistics"
+STATS_REDIS_KEY = "recent-stats"
+STATS_REDIS_LAST_UPDATE_KEY = "recent-stats-last-updated"
+STATS_REDIS_NAMESPACE = "statistics"
```

Comment:
So then something like:

`STATS_CACHE_KEY`
`STATS_CACHE_LAST_UPDATE_KEY`
`STATS_CACHE_NAMESPACE`

^this?